modules/shared/atoms/Button/IButton.d.ts

```diff
@@ -0,0 +1,13 @@
+declare namespace IButton {
+  export interface IProps {
+    children?: string;
+    disabled?: false | true;
+    variant?: 'primary' | 'secondary' | 'text';
+    size?: 'lg' | 'md' | 'sm';
+    leftIcon?: false | true;
+    rightIcon?: false | true;
+    onlyIcon?: false | true;
+    mockFunction?: () => void;
```

Comment:
The function name must reflect its specific action. I think that this name is taken from the test, and the code should not be changed because of the test.